Patch by Sergei Golovan allows to find tclConfig.sh in /usr/lib/tcl8.6
and tkConfig.sh in /usr/lib/tk8.6 where they are located in Debian
installation.

--- tcl8.6-8.6.0.orig/unix/tcl.m4
+++ tcl8.6-8.6.0/unix/tcl.m4
@@ -93,6 +93,7 @@
 			`ls -d ${prefix}/lib 2>/dev/null` \
 			`ls -d /usr/local/lib 2>/dev/null` \
 			`ls -d /usr/contrib/lib 2>/dev/null` \
+			`ls -d /usr/lib/tcl8.6 2>/dev/null` \
 			`ls -d /usr/lib 2>/dev/null` \
 			`ls -d /usr/lib64 2>/dev/null` \
 			; do
@@ -223,6 +224,7 @@
 			`ls -d ${prefix}/lib 2>/dev/null` \
 			`ls -d /usr/local/lib 2>/dev/null` \
 			`ls -d /usr/contrib/lib 2>/dev/null` \
+			`ls -d /usr/lib/tk8.6 2>/dev/null` \
 			`ls -d /usr/lib 2>/dev/null` \
 			`ls -d /usr/lib64 2>/dev/null` \
 			; do
